data/error_policies/parent_call.cas

```diff
@@ -0,0 +1,15 @@
+virtual resource bar {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+resource foo inherits bar {
+    fn read(domain source) {
```

Comment:
This line and line 14 have spaces, while the rest of the rest of the file has tabs.